phpstan.neon

```diff
@@ -0,0 +1,25 @@
+parameters:
+    autoload_directories:
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/ClassLoaderTest
+    autoload_files:
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Persistence/PersistentObjectTest.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Persistence/Mapping/ClassMetadataFactoryTest.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Persistence/Mapping/_files/annotation/TestClass.php
+    excludes_analyse:
+        - %currentWorkingDirectory%/lib/vendor/doctrine-build-common
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Proxy/InvalidReturnTypeClass.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Proxy/InvalidTypeHintClass.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Persistence/Mapping/_files/TestEntity.php
+        - %currentWorkingDirectory%/tests/Doctrine/Tests/Common/Proxy/generated
+    ignoreErrors:
+        - '#Doctrine\\Common\\Reflection\\StaticReflection[a-zA-Z0-9_]+::__construct\(\) does not call parent constructor from Reflection[a-zA-Z0-9_]+#'
+        - '#\(void\) is used#'
+        - '#Call to an undefined method PHPUnit_Framework_MockObject_MockObject::#'
+        - '#Call to an undefined method ReflectionType::getName\(\)#'
+        - '#Call to an undefined method Doctrine\\Tests\\Common\\Persistence\\TestObject#'
+        - '#with return type void returns void but should not return anything#'
+        - '#does not accept PHPUnit_Framework_MockObject_MockObject#'
+        - '#Call to an undefined method Doctrine\\Tests\\Common\\Proxy\\LazyLoadableObject[a-zA-Z0-9_]*::__[a-zA-Z0-9_]+\(\)#'
+        - '#Access to an undefined property Doctrine\\Common\\Proxy\\Proxy::\$publicField#'
```

Comment:
This is from this test:

```
 ------ ----------------------------------------------------------------------------
  Line   tests/Doctrine/Tests/Common/Proxy/ProxyMagicMethodsTest.php
 ------ ----------------------------------------------------------------------------
  82     Access to an undefined property Doctrine\Common\Proxy\Proxy::$publicField.
  181    Access to an undefined property Doctrine\Common\Proxy\Proxy::$publicField.
  196    Access to an undefined property Doctrine\Common\Proxy\Proxy::$publicField.
 ------ ----------------------------------------------------------------------------
```

I don't see how that could be hinted, it's about this concrete instance.